elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```
@@ -0,0 +1,324 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: jump;
+
+$variables: (
+  Padding: calc(#{pfe-var(container-spacer)} * 2),
+  BackgroundColor: pfe-color(surface--base),
+);
+
+:host {
+  //Default styles
+  // font-family: 'Red Hat Text', sans-serif;
+  // @include pfe-set-broadcast-theme(light);
+  // --theme: light;
+
+  // Example local variable, using BEM format
+  // --pfe-jump-links--Property: value;
+  font-weight: 400;
+  display: block;
+  position: sticky;
+  top: calc(#{pfe-var(container-spacer)} * 4);
+  padding: calc(#{pfe-var(container-spacer)} * 2);
+  --pfe-jump-links--BackgroundColor: BackgroundColor;
```

I'm not sure what this is supposed to be doing :) maybe it should be this?
```suggestion
  background-color: #{pfe-local(BackgroundColor)};
```